apps/compound/src/components/WidgetWrapper.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+type Props = {
+  children: React.ReactNode;
+};
+
+export default function WidgetWrapper({ children }: Props): React.ReactElement {
```

Comment:
nitpick and a tip: if you use React.FC as a component type, it already includes type for `children`.

I'd say it's better to use arrow functions + React.FC just to stay consistent with how components are defined within the project.

Reply:
nice!